### ISSUE TITLE
UI: 2.2 - Prometheus error when installing Monitoring 

### ISSUE THREAD

**Issue by loganhz** [rancher/rancher#19197]
UI for https://github.com/rancher/rancher/issues/19278

**Comment by cjellick** [comment]
@loganhz can you attach a screenshot of what this looks like when monitoring is in a bad state in 2.2.0? we want to know how bad it is to determine if we need to fix for 2.2.2

**Comment by loganhz** [comment]
https://github.com/rancher/rancher/issues/19068#issuecomment-475463866

Something like this.

**Comment by davidnuzik** [comment]
This was also previously tested in v2.2.2-rc2 and rc5 via #19326 I checked again in latest master for v2.3
Version: Master (4/15/19)

I followed the steps to test outlined here: https://github.com/rancher/rancher/issues/19326#issuecomment-479814459

1. Enable Cluster Monitoring -> the monitoring dashboards should be on
2. Disable Cluster Monitoring -> the monitoring dashboards should be off
3. Enable Cluster Monitoring and then drop `cattle-prometheus/prometheus-cluster-monitoring` Pod -> the monitoring dashboards should be off until Prometheus Pod redeploy

I confirmed that this works (step 2 and 3 were the fix) the dashboard instantly goes to the classic cluster page when we disable monitoring or delete the prometheus-cluster-monitoring-0 pod. It then quickly recovers and brings a new pod up.

After up to about 1-2 mins (with my testing) the cluster dashboard page will show the monitoring page again. When you manually delete the whole prometheus-cluster-monnitoring-0 pod you will see an error in the UI but monitoring does eventually come back with a manual refresh. The important thing is we fall back to normal cluster dashboard view when monitoring is not healthy.
